docs/user/rollout-policies.md

```diff
@@ -0,0 +1,123 @@
+# Rollout Device Selection and Rollout Disruption Budget
```

Comment:
The reference in `user/README.md` to the definition of the rollout policies needs to be updated.

Currently it links to `managing-fleets.md#defining-rollout-policies` which does not exist, and in this PR it should point to `rollout-policies.md`